contracts/Scroll_SpokePool.sol

```diff
@@ -12,11 +19,13 @@ import "@scroll-tech/contracts/libraries/IScrollMessenger.sol";
  * @custom:security-contact bugs@across.to
  */
 contract Scroll_SpokePool is SpokePool {
+    using SafeERC20 for IERC20;
```

Comment:
Should instead use SafeERC20Upgradeable.

Reply:
This is inherited already, no?

Reply:
It's doesn't appear to be automatically inherited. We do this in some other contracts, though we're not consistent about the use of SafeERC20Upgradeable vs. SafeERC20. I've followed the Ethereum SpokePool: https://github.com/across-protocol/contracts/blob/1471664a516f0d81632f7228eeb90afcf350236a/contracts/Ethereum_SpokePool.sol#L12

We use SafeERC20 for the Linea and OVM SpokePools. That however requires additional imports, so I followed the Ethereum SpokePool.

https://github.com/across-protocol/contracts/blob/1471664a516f0d81632f7228eeb90afcf350236a/contracts/Linea_SpokePool.sol#L17
https://github.com/across-protocol/contracts/blob/1471664a516f0d81632f7228eeb90afcf350236a/contracts/Ovm_SpokePool.sol#L36